modules/standard/Path.chpl

```diff
@@ -260,25 +267,29 @@ proc commonPath(paths: []): string {
     delimiter = "\\";
   }
 
-  var prefixArray = firstPath.split(delimiter, -1, false);
+  var prefixList = new list(string);
+  for x in firstPath.split(delimiter, -1, false) do
+    prefixList.append(x);
```

Comment:
Same comments about variable naming and initialization. Variable naming is much of a matter of taste, so up to you.